### PR TITLE
Create VPC

### PR DESCRIPTION
**Pull request by jhash1**

### CODE REVIEW

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Please do the following:
- Rename this pull request. We need the name that describes what exactly changed. For example "Configure a VPC"
- Remove db.tf this code is not relevant to this task. It's a bad practice to mix multiple irrelevant changes in a single PR

File: variables.tf [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+# variable "aws_ami" {
+#   description = "default ami for wordpress"
+#   default     = "ami-08704011bfd36e43b"
+# } 
+
+# variable "db_password" {
+#   description = "Database administrator password"
+#   type        = string
+#   sensitive   = true
+# }
```
Comment (vladimirmukhin) [inline variables.tf:10]: Please don't leave commented-out code blocks. It's a bad style practice

File: variables.tf [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+
+variable env_code {}
+
+variable "vpc_cidr" {}
+
+variable "private_cidr" {}
+
+variable "public_cidr" {}
```
Comment (vladimirmukhin) [inline variables.tf:18]: Please always add an empty newline to the end of file.
Linux tools and systems expect a newline at the end of file and might work not correctly

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Not all resources are tagged. Please check your code and add missing tags

File: variables.tf (hunk 2 above)
Comment (jhash1) [reply variables.tf]: all set

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**